examples/with-angular-thirdpartyemailpassword/server.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express");
```

Comment:
Convert this project to use TS

Reply:
For the server, you can run it using ts-node (instead of node), so you should add ts-node as a packaege.json dependency as well.

ts-node allows you to directly run ts files for the server